modules/core/shared/src/main/scala-3/io/circe/derivation/Default.scala

```diff
@@ -0,0 +1,46 @@
+package io.circe.derivation
+
+import scala.quoted.*
+import scala.deriving.*
+import scala.compiletime.constValue
+
+/**
+  * Original code by Dmytro Mitin, with slight modifications by Simão Martins.
+  * See: https://stackoverflow.com/questions/68421043/type-class-derivation-accessing-default-values
+  */
+
+trait Default[T]:
```

Comment:
```suggestion
trait Default[T] extends Serializable:
```

Reply:
I don't understand why `Default` needs to be serializable.

Reply:
`Encoder` and `Decoder` traits are both `Serializable`. Circe test set ensures that is true. As `Default` is declared as a field of `ConfiguredDecoder` it makes it non-serializable. https://github.com/circe/circe/pull/1800/files#diff-378e58dd77c3ec12a0e0b230ec5d55a15679e08f37f01ef83c5ee06967ca42d4R143

Reply:
There might be another way of making `ConfiguredDecoder` serializable of course.